ios-target/Networking/Services/TargetAPI.swift

```diff
@@ -55,4 +55,25 @@ class TargetAPI {
       }
     })
   }
+  
+  // TODO: pending feature for missing backend
```

Comment:
instead of commenting out the code you don't want in the PR, you can commit without this file and commit it when you are ready to use it.